.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9, "3.10"]
+        python-version: [3.7, 3.8, 3.9, "3.10"]
```

Comment:
I wonder if the quotes around 3.10 are necessary. But you didn't do it, so it does not matter so much.

Reply:
It does, it is a common quirk of yaml, `3.10` will be parsed as `3.1`(float)